common/app/redux/reducer.js

```diff
@@ -16,11 +16,15 @@ const initialState = {
 
 export default handleActions(
   {
-    [types.updateTitle]: (state, { payload = 'Learn To Code' }) => ({
+    [types.updateTitle]:
+    (state,
```

Comment:
style-args: Function args should either be all inline or each on it's own line
style-object-spacing: Curly braces should be padded with a space.

```js
[types.updateTitle]: (
  state,
  { payload: { title = 'Learn to Code', blockType = 'Free Code Camp' } }
) => {
// ....
```